src/controllers/lucky.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from "express";
+import * as Services from "../dal/lucky";
+import { getText } from "../helpers/arrayProcessing";
+import { getTodayAllSignRecordsFromDB } from "../dal/zodiacdeep";
+
+export async function getLuckyTextByRandomNumber(
+  req: Request,
+  res: Response
+): Promise<void> {
+  {
+    const luckyText = req?.query?.lucky;
+    let totalResult = "";
+    if (typeof luckyText !== "undefined" && luckyText === "lucky") {
+      if (luckyText) {
+        totalResult = await Services.getLuckyTextByRandomNumber();
+      }
+    } else {
+      const resultTextArray = await getTodayAllSignRecordsFromDB();
+      if (resultTextArray) {
+        totalResult = getText(resultTextArray);
+      } else {
+        console.log("Not ok getting from db..");
```

Comment:
This text is not saying much - you can replace it with the standard ERROR 404 Not found test and instead of logging it you can return it to the user.

Reply:
done